osu.Game/Overlays/TabControlOverlayHeader.cs

```diff
@@ -54,10 +75,10 @@ protected override TabItem<string> CreateTabItem(string value) => new OverlayHea
 
             private class OverlayHeaderTabItem : OverlayTabItem
             {
-                public OverlayHeaderTabItem(string value)
+                public OverlayHeaderTabItem(T value)
                     : base(value)
                 {
-                    Text.Text = value;
+                    Text.Text = value.ToString().ToLowerInvariant();
```

Comment:
Not sure about the invariance in the long haul (localisation is what I'm worried about).